api/src/test/java/org/openmrs/ObsTest.java

```diff
@@ -1086,4 +1098,16 @@ public void setValueBoolean_shouldNotSetValueForNonBooleanConcept() throws Excep
 		obs.setValueBoolean(null);
 		assertNotNull(obs.getValueCoded());
 	}
+	@Test
+	public void removingObsFromObsGroup_shouldRemoveChildObs() throws Exception {
```

Comment:
This test passes well locally though  i am getting a null exception for a couple of different tests cc @ibacher